include/extractor/guidance/turn_instruction.hpp

```diff
@@ -43,8 +43,14 @@ enum TurnType // at the moment we can support 32 turn types, without increasing
     NewName,                // no turn, but name changes
```

Comment:
Can we merge `NewName` and `Notification`? `Notification` just tells the user that a street attribute (like name or travel mode) changed. This would be forward compatible if we add more attributes for which we might want to emit a `Notification.